litmus-portal/namespaced-K8s-manifest.yml

```diff
@@ -0,0 +1,323 @@
+---
+apiVersion: v1
```

Comment:
Should the namespace creation be removed from namespaced-manifest? @rajdas98

Reply:
Actually, I can add it, but it would be redundant @gdsoumya ...

Reply:
I am not talking about adding, I am talking about removing it because the user might not have access to creating a ns

Reply:
Yeah, then it can be removed @gdsoumya too. cc: @rajdas98